android/sdl_android/src/main/java/com/smartdevicelink/encoder/VirtualDisplayEncoder.java

```diff
@@ -203,9 +204,15 @@ public void run() {
                     inputSurface = prepareVideoEncoder();
 
                     // Create a virtual display that will output to our encoder.
+                    if (Build.VERSION.SDK_INT >= VERSION_CODES.LOLLIPOP && virtualDisplay != null) {
+                        virtualDisplay.setSurface(inputSurface);
+                    }
+                else {
```

Comment:
Fix the formatting of this else statement so it is inline with the correct if statement